include/galsim/SBInclinedExponentialImpl.h

```diff
@@ -0,0 +1,218 @@
+/* -*- c++ -*-
+ * Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+ * https://github.com/GalSim-developers
+ *
+ * This file is part of GalSim: The modular galaxy image simulation toolkit.
+ * https://github.com/GalSim-developers/GalSim
+ *
+ * GalSim is free software: redistribution and use in source and binary forms,
+ * with or without modification, are permitted provided that the following
+ * conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions, and the disclaimer given in the accompanying LICENSE
+ *    file.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions, and the disclaimer given in the documentation
+ *    and/or other materials provided with the distribution.
+ */
+
+#ifndef GalSim_SBInclinedExponentialImpl_H
+#define GalSim_SBInclinedExponentialImpl_H
+
+#include "SBProfileImpl.h"
+#include "SBInclinedExponential.h"
+#include "LRUCache.h"
+#include "Table.h"
+
+namespace galsim {
+
+    /// @brief A private class that caches the needed parameters for each inclined exponential angle 'i'
+    class InclinedExponentialInfo
+    {
```

Comment:
I don't see any reason it would be useful for photon shooting.  You need a different IFT for each object don't you?  The Info classes are intended for calculations that are common to all or many objects.  e.g. for Sersic there are a bunch of calculations that need to be done once for each value of n.  Since many applications repeat the same n many times, it is useful to cache these values once.

As far as I can tell, the InclinedExponential calculations need to be redone for each value of h_tani_over_r, and there is no reason to expect these to be repeated very often in typical use cases.